Assets/Scripts/Plant.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Burst.CompilerServices;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+[CreateAssetMenu(menuName = "Plants/base plant", fileName = "NewPlantData")]
+public class Plant : ScriptableObject
+{
+    //Potencjalne problemy do omowienia:
+    //>Co z grafikami? Kopiujac zajmujemy wiecej pamieci? Moze klasa PlantData
+    //przechowujaca je w jednym miejscu caly czas? Ale jak ja wtedy zapisac? ID? NameID?
```

Comment:
Grafikami? A nie jest to konieczne żeby każda grafika po prostu raz istniała gdzieś i była przypisana do danego SO swojego? W sensie że np jak jest teraz ten "base plant" to żeby tak wszystkie roślinki dodawać i tam umieszczać jakoś grafiki żeby one potem były jakoś ściągane z tego tylko? Brzmi to możliwie i logicznie?

Reply:
Brzmi bardzo logicznie. Zajmę się implementacją :)